### PR TITLE
PDF line numbers for new linenumbering.js (fixes #2441)

### LINKED ISSUE #2441: Text with line numbers is not shown in motion pdf
How to reproduce this regression:
* Create a new motion with example text.
* Create pdf of this motion (without line numbers) -> Ok.
* Switch to line numbering mode inline or outside.
* Create pdf again (with line numbers) -> PDF contains no motion text! 

I need a fix today!

### PR DESCRIPTION
**Pull request by tsiegleauq**
fixes the issue mentioned above.
No examples this time.

Thanks to @FinnStutzenstein for helping out with the required information!

### CODE REVIEW

File: openslides/motions/static/js/motions/site.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions', 'OpenSlid
                 * other project that might want to use this HTML to PDF parser.
                 * https://github.com/OpenSlides/OpenSlides/issues/2361
                 */
-                return converter.convertHTML($scope.lineBrokenText, $scope);
+                return converter.convertHTML(motion.getTextWithLineBreaks($scope.version, 0), $scope);
```
Comment (normanjaeckel) [inline openslides/motions/static/js/motions/site.js]: What is this `0` over there? Does this mean you use always the first version of possbily many existing?

Comment (tsiegleauq) [inline openslides/motions/static/js/motions/site.js]: from
https://github.com/OpenSlides/OpenSlides/blob/master/openslides/motions/static/js/motions/base.js#L184

```
getTextWithLineBreaks: function (versionId, highlight, callback) {
    var lineLength = Config.get('motions_line_length').value,
    html = this.getVersion(versionId).text;

    return lineNumberingService.insertLineNumbers(html, lineLength, highlight, callback);
},
```
We use the given `$scope.version` and no (`0`) highlighting and no callback function

Comment (FinnStutzenstein) [inline openslides/motions/static/js/motions/site.js]: @tsiegleauq You could maybe just test if you can pass undefined. I don't know how the software behaves, but you may just write a check for this [here](https://github.com/OpenSlides/OpenSlides/blob/master/openslides/motions/static/js/motions/linenumbering.js#L89) where the highlighting is applied. Something like:
```
if (typeof highlight === 'undefined') {
highlight=0;
}
```

**Comment by emanuelschuetze** [comment]
@normanjaeckel As discussed I added a new commit to this pr. Please review and merge.

**Pull request merged**